src/main/java/uk/gov/companieshouse/itemgroupworkflowapi/controller/ItemGroupController.java

```diff
@@ -13,96 +15,119 @@
+    public ItemGroupController(LoggingUtils logger,
+                               ItemGroupsService itemGroupsService,
+                               ItemGroupsValidator itemGroupsValidator) {
         this.logger = logger;
         this.itemGroupsService = itemGroupsService;
         this.itemGroupsValidator = itemGroupsValidator;
     }
-
-    @PostMapping("${uk.gov.companieshouse.itemgroupworkflowapi.createitemgroup}")
-    public ResponseEntity<Object> createItemGroup(final @RequestHeader(REQUEST_ID_HEADER_NAME) String requestId,
+    /**
+     * Create item group from ItemGroupData<p>
+     * POST RequestID logged<p>
+     * Validation performed on item group, on failure return BAD_REQUEST<p>
+     * Item group already exists check performed, if already exists return CONFLICT<p>
+     * Item group created, on success return CREATED, on error return BAD_REQUEST<p>
+     */
+    @PostMapping(path = "${uk.gov.companieshouse.itemgroupworkflowapi.createitemgroup}",
+            consumes = MediaType.APPLICATION_JSON_VALUE,
+            produces = MediaType.APPLICATION_JSON_VALUE)
+    public ResponseEntity<Object> createItemGroup(final @RequestHeader(X_REQUEST_ID_HEADER_NAME) String xRequestId,
                                                   final @RequestBody ItemGroupData itemGroupData) {
-        logRequestId(requestId);
-        List<String> errors = itemGroupsValidator.validateCreateItemPayload(itemGroupData);
+        logRequestId(xRequestId);
+        List<String> errors = itemGroupsValidator.validateCreateItemData(itemGroupData);
 
         if (!errors.isEmpty()) {
-            return buildValidationResponse(errors, itemGroupData);
+            return buildValidationResponse(xRequestId, errors);
         }
 
         try {
-            if (itemGroupsService.doesItemGroupExist(itemGroupData)) {
-                return buildItemAlreadyExistsResponse(itemGroupData);
-            }
+            if (itemGroupsService.doesItemGroupExist(itemGroupData))
+                return buildItemAlreadyExistsResponse(xRequestId, itemGroupData);
 
             final ItemGroup savedItem = itemGroupsService.createItemGroup(itemGroupData);
-            return buildCreateSuccessResponse(savedItem);
-
-        } catch (Exception ex) {
-            return buildErrorResponse(ex, itemGroupData);
+            return buildCreateSuccessResponse(xRequestId, savedItem);
+        }
+        catch (Exception ex) {
+            return buildErrorResponse(xRequestId, ex, itemGroupData);
         }
     }
 
-    private void logRequestId(String requestId) {
-        Map<String, Object> logMap = logger.createLogMap();
-        logMap.put(REQUEST_ID_LOG_KEY, requestId);
-        logger.getLogger().info("create item group: request id", logMap);
-    }
+    private void logRequestId(String xRequestId) {
+        DataMap dataMap = new DataMap.Builder()
+            .xRequestId(xRequestId)
```

Comment:
Following discussions, we have removed this `xRequestId()` method from the DataMap Builder as a duplicate key logging key for the request ID (just now).